polaris-tokens/package.json

```diff
@@ -4,12 +4,11 @@
   "description": "",
   "main": "dist/index.js",
   "scripts": {
-    "build": "tsc -b",
-    "test": "yarn validate-tokens",
-    "validate-tokens": "ajv validate -s schemas/token-group.json -d 'token-groups/*.json'"
+    "build": "tsc -b && node scripts/toJSON && node scripts/toStyleSheet",
```

Comment:
Should we just have a single `build-tokens` script that runs both of these functions?